docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -427,7 +427,9 @@ POST api/detection_engine/rules/_bulk_action
 [discrete]
 ===== Response payload
 
-For `enable`, `disable`, `delete`, `edit`, and `duplicate` actions, a JSON object containing the action's outcome: number of failed and successful outcomes, number of rules, and number of rule objects created, deleted or updated (depending on the action).
+For `enable`, `disable`, `delete`, `edit`, and `duplicate` actions, a JSON object containing the action's outcome: number of failed and successful outcomes, number of rules, and number of rule objects created, deleted, skipped or updated (depending on the action).
+
+Rules can only be `skipped` when performing an  `edit` action, when all the operations to be performed on any one rule result in a no-op. For example, if the `edit` action is to add a tag to a rule that already has that tag, and to delete an index pattern that doesn't exist. Skipped rules objects will only return their `id`, `name`, and `skip_reason`.
```

Comment:
"No-op" is a bit of jargon, so this edit tries a simpler explanation — please confirm if it's accurate, as it's a new term for me.

Also a little uncertain about "and" or "or" with the two examples. I _think_ they're just two separate examples, so "or" would make sense, but if the two examples are actually connected and have to function as a combination of actions, then "and" is probably best.
```suggestion
Rules can only be `skipped` when performing an `edit` action, when all the operations to be performed on any one rule result in nothing being done. For example, if the `edit` action is to add a tag to a rule that already has that tag, or to delete an index pattern that doesn't exist. Skipped rules objects will only return their `id`, `name`, and `skip_reason`.
```

Reply:
I agree with @joepeeples suggestions regarding the noop term and the "and vs or". I think "or" would be easier to grasp for the user. @joepeeples FYI it could be "and" as well because the API supports sending multiple bulk edit operations in a single request.

I'd simplify it a bit further:

```suggestion
A rule can only be `skipped` when the bulk action to be performed on it results in nothing being done. For example, if the `edit` action is used to add a tag to a rule that already has that tag, or to delete an index pattern that is not specified in a rule. Objects returned in `attributes.results.skipped` will only include rules' `id`, `name`, and `skip_reason`.
```

Not mentioning the `Rules can only be skipped when performing an edit action` gives us the freedom to expand skipping to other cases.